mongodb/spring-cloud-starter-stream-source-mongodb/src/main/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+collection=activationRecord
+query={ }
+uri=mongodb://localhost:27017/test
```

Comment:
No any `application.properties` in the `main`.
Just because they must be specified externally, e.g. via SCDF